pkg/apis/flowcontrol/types.go

```diff
@@ -232,6 +237,17 @@ type ResourcePolicyRule struct {
 	// +listType=set
 	// Required.
 	Resources []string
+	// `namespaces` is a list of target namespaces that restricts
+	// matches.  A request that does not specify a target namespace
+	// (which happens both when the resource is not namespaced and
+	// when the resource is namespaced and the request is for all
+	// namespaces) matches only if this list includes "Cluster Scope".
```

Comment:
I'd insert `(a string literal which is not a valid namespace name and therefore can't be confused with a namespace)` or something to this effect.